src/tools/wasm-merge.cpp

```diff
@@ -122,15 +122,15 @@ struct Mergeable {
     findImportsByBase(wasm, MEMORY_BASE, [&](Name name) {
       memoryBaseGlobals.insert(name);
     });
-    if (memoryBaseGlobals.size() == 0) {
-      Fatal() << "no memory base was imported";
-    }
+    //if (memoryBaseGlobals.size() == 0) {
```

Comment:
Removing and yes tests are needed, my current example was combing two separately compiled C modules, what I was thinking was taking the wast output of those and build a test mimicking the current tests, is that acceptable?